packages/api/src/derives/staking/index.ts

```diff
@@ -1 +1,11 @@
-export {queryStakingAccountDetails as accountInfo, querySession as sessionDetails} from './stakingAccount';
+export {
+  queryStakingAccountDetails as accountInfo,
+  querySession as sessionDetails,
```

Comment:
Nit: a bit more consistency with naming would be good here
```suggestion
  queryStakingAccountInfo as accountInfo,
  querySessionInfo as sessionKeyInfo,
```

Reply:
done